### PR TITLE
Make SC great for the first time

### PR DESCRIPTION
**Pull request by parostatkiem**
**Description**

Changes proposed in this pull request:

- Create a brand new _create-react-app_ application
- put all service-catalog views into it
- use common makefile
- use only `LuigiClient.getContext()` to get the namespace ID
- merge all "commons", "shared" and other similar files, then divide it in a more natural way
- skip merging `testing` folder because it's pointless
- move `ModalWithForm` to _Shared_
- set Nginx port to 80

**Related issue(s)**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +21,7 @@ The Console also includes React and Angular libraries:
 - [`React common`](./common) - common functionalities for React applications
 - [`React components`](./components/react) - components for React applications (it will be replaced by `Shared components`)
 - [`Shared components`](./components/shared) - new versions of components for React applications written in TypeScript
-- [`Generic documentation`](./components/generic-documentation) - a React component that uses [`@kyma-project/documentation-component`](https://github.com/kyma-incubator/documentation-component) for displaying documentation and various specifications in the [`Content`](./content), [`Catalog`](./service-catalog-ui/catalog) and [`Instances`](./service-catalog-ui/instances) views.
+- [`Generic documentation`](./components/generic-documentation) - a React component that uses [`@kyma-project/documentation-component`](https://github.com/kyma-incubator/documentation-component) for displaying documentation and various specifications in the [`Content`](./content) and [`Service-Catalog-UI`](./service-catalog-ui) views.
```
[the remark this answers is not in the document]
Comment (parostatkiem) [reply README.md:24]: What is `cockpit-ui`? You mean Compass?

[the remark this answers is not in the document]
Comment (akucharska) [reply README.md:24]: yes

File: service-catalog-ui/README.md [hunk 2]
```diff
@@ -1,48 +1,68 @@
...
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
+
+### Deployment
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
+
+### `npm run build` fails to minify
+
+This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
```
Comment (bszwarc) [inline service-catalog-ui/README.md:68]:
```suggestion
### Code splitting

This section was moved here: https://facebook.github.io/create-react-app/docs/code-splitting

### Analyzing the bundle size

This section was moved here: https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size

### Making a progressive web app

This section was moved here: https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app

### Advanced configuration

This section was moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration

### Deployment

This section was moved here: https://facebook.github.io/create-react-app/docs/deployment

### Troubleshooting

For troubleshooting to `npm run build` fails to minify error, see https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
```
Maybe this section would look better as a list of links? or a table? 🤔  What do you think?

Comment (parostatkiem) [reply service-catalog-ui/README.md:68]: I think it looks perfect the way you suggested 💎

**Pull request merged**